spec/hbw/features/cancel_process_button_mock.yml

```diff
@@ -0,0 +1,194 @@
+
+get:
+  tasks: {}
+  user:
+    - params: "email=c.johnson@example.com"
+      response:
+        data:
+          - id:         'demo'
+            firstName:  'Demo'
+            lastName:   'Demo'
+            url:        'http://bpm.example.com'
+            email:      'c.johnson@example.com'
+            pictureUrl: null
+        total: 1
+        start: 0
+        sort: 'id'
+        order: 'asc'
+        size: 1
+
+  # process definitions
+  process-definition:
+    - params: ""
+      response:
+        - &default_process-definition
+          id: "testProcess:469:222221"
+          key: "supportRequestProcess"
+          version: 469
+          name: "Test name"
+          description: null
+          tenantId: null
+          deploymentId: "222221"
+          resource: "testProcess.bpmn"
+          diagram: null
+          category: "http://bpmn.io/schema/bpmn"
+          suspended: false
+          versionTag: null
+          historyTimeToLive: null
+
+  # form definitions
+  task/111111/deployed-form:
+    - params: ""
+      response: |
+        form:
+          name: Test form with file upload without file list
+          css_class: col-xs-12 col-sm-6 col-md-5 col-lg-4
+          fields:
+          - name: group1
+            type: group
+            label: Files
+            css_class: col-xs-12
+            fields:
+            - name: multipleFileUpload
+              type: file_upload
+              label: Attach multiple files
+              file_list_name: FL_MultipleFileUpload
+              css_class: col-md-4 col-xs-4 col-lg-4
+              multiple: true
+            - name: FL_MultipleFileUpload
+              type: file_list
+              label: Attached files
+              css_class: col-md-8 col-xs-8 col-lg-8
+
+            - name: paramAbsentFileUpload
+              type: file_upload
+              label: Attach single file in group
+              css_class: col-md-4 col-xs-4 col-lg-4
+              input_text: ''
+            - name: homsOrderDataFileList
+              type: file_list
+              label: Attached files
+              css_class: col-md-8 col-xs-8 col-lg-8
+
+            - name: emptyFileUpload
+              type: file_upload
+              label: Attach single file in group
+              file_list_name: FL_EmptyFileUpload
+              css_class: col-md-4 col-xs-4 col-lg-4
+              input_text: ''
+            - name: FL_EmptyFileUpload
+              type: file_list
+              label: Attached files
+              css_class: col-md-8 col-xs-8 col-lg-8
+
+          - name: singleFileUpload
+            type: file_upload
+            label: Attach single file
+            file_list_name: FL_SingleFileUpload
+            css_class: col-md-4 col-xs-4 col-lg-4
+            input_text: 'Drag single file, or '
+          - name: FL_SingleFileUpload
+            type: file_list
+            label: Attached files
+            css_class: col-md-8 col-xs-8 col-lg-8
```

Comment:
This test definitely does not need all these fields.

Reply:
I'll make issue about cutting off such redundant things from tests